webonary-cloud-api/lambda/__tests__/databaseSetup.ts

```diff
@@ -32,3 +31,12 @@ export async function createDictionary(data = '{}'): Promise<string> {
   await upsertDictionary(data, dictionaryId, 'test-user');
   return dictionaryId;
 }
+
+export function parseGuids(response: APIGatewayProxyResult): string[] {
```

Comment:
nit: This doesn't have anything to do with databases. Perhaps it'll be a better fit in some kind of kitchen sink utils file?

Reply:
yes, you're right. thanks.
created a utils fils, and moved it there.